### DISCUSSION TITLE
Settings Web Page

### DISCUSSION THREAD

**Discussion by thesteward221b** [Tzahi12345/YoutubeDL-Material#853] [Q&A]
This is probably answered somewhere so I apologize, but I couldn't find it.

We're testing the Youtube-DL program and I've been very impressed so far!  One thing I noticed though is that the Settings page is available for anyone to access and make changes; is there a way to remove that or lock it down without enabling multiuser mode?

**Comment by Tzahi12345** [comment]
Have you enabled multi-user mode? It should be locked out to non admins.

**Reply by thesteward221b** [reply]
Yeah, I did try that out and it would address that issue, but I'm trying to keep the process simple for simple people.  I figured out a way to keep the config in check though, so it's good

**Reply by Tzahi12345** [reply]
I see, I guess what you're looking for something like what this issue describes? https://github.com/Tzahi12345/YoutubeDL-Material/issues/545

I guess just for data points sake, what's your use case here? If you don't mind sharing.

**Reply by thesteward221b** [reply]
Oh yeah, that pin system would work well for what we're needing without adding complication for our users.

I probably over-engineered this, but I set up a powershell DSC config that compares the contents of the default.json file against a master copy housed on another server, if the two files don't match then it updates the in use copy.  Doesn't stop anyone from changing settings but it makes reverting the changes automatic.
